npx ignite-cli new fails

### Describe the bug

The following error occurred:
    
    Git status is dirty but the command will continue because the terminal is not interactive.
CommandError: Error reading Expo config at /Users/dkc/src/iPhone/PizzaApp/app.config.ts:

⨯ Unable to compile TypeScript:
error TS6053: File 'expo/tsconfig.base' not found.


error Command failed with exit code 1.


i'm quite willing to believe it is me and not the software.  but thanks.

### Ignite version

9.0.1

### Additional info

System
  platform           darwin                                                
  arch               x64                                                   
  cpu                4 cores      Intel(R) Core(TM) i5-5350U CPU @ 1.80GHz 
  directory          PizzaApp     /Users/dkc/src/iPhone/PizzaApp           

JavaScript (and globally-installed packages)
  node                   20.9.0       /usr/local/bin/node 
  npm                    10.1.0       /usr/local/bin/npm  
    corepack             0.20.0                           
    dependency-cruiser   15.3.0                           
    npm                  10.1.0                           
  yarn                   1.22.19      /usr/local/bin/yarn 
    node-gyp             9.4.0                            
    react-native-cli     2.0.1                            
  pnpm                   -            not installed       
  bun                    -            not installed       
  expo                   49.0.19      managed             

Ignite
  ignite-cli         9.0.1        /Users/dkc/.npm/_npx/e31027f3785124a8/node_modules/.bin/ignite      
  ignite src         build        /Users/dkc/.npm/_npx/e31027f3785124a8/node_modules/ignite-cli/build 

Android
  java               11.0.16.1    /usr/bin/java 
  android home       -            undefined     

iOS
  xcode              14.2       
  cocoapods          1.12.1       /usr/local/bin/pod 

Tools
  git                git version 2.37.1 (Apple Git-137.1)   /usr/bin/git

Same here. I updated and it broke so I made a new project and it failed as well. 

```
❯ npm run android

> yolda-app@0.0.1 android
> npx expo start --android

Starting project at C:\CODING\LEARNING\MOBILEAPPS\YoldaApp
TSError: Error reading Expo config at C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\app.config.ts:

⨯ Unable to compile TypeScript:
error TS6053: File 'expo/tsconfig.base' not found.

TSError: Error reading Expo config at C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\app.config.ts:

⨯ Unable to compile TypeScript:
error TS6053: File 'expo/tsconfig.base' not found.

    at createTSError (C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\node_modules\ts-node\src\index.ts:859:12)
    at reportTSError (C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\node_modules\ts-node\src\index.ts:863:19)
    at createFromPreloadedConfig (C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\node_modules\ts-node\src\index.ts:874:36)
    at create (C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\node_modules\ts-node\src\index.ts:624:10)
    at Object.register (C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\node_modules\ts-node\src\index.ts:591:15)
    at Object.<anonymous> (C:\CODING\LEARNING\MOBILEAPPS\YoldaApp\node_modules\ts-node\register\index.js:1:16)
    at Module._compile (node:internal/modules/cjs/loader:1241:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Function.Module._load (node:internal/modules/cjs/loader:938:12)
```



### Ignite version

9.0.0

### Additional info

```
 platform           win32
  arch               x64
  cpu                12 cores     AMD Ryzen 5 5600X 6-Core Processor
  directory          PizzaApp3    C:\CODING\LEARNING\MOBILE APPS\PizzaApp3

```

--- a/boilerplate/package.json
+++ b/boilerplate/package.json
@@ -100,7 +100,7 @@
     "regenerator-runtime": "^0.13.4",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.1",
-    "typescript": "^5.2.2"
+    "typescript": "5.2.2"
   },
   "expo": {
     "install": {

